File: app/src/main/java/com/example/sportapp/support2026/features/news/domain/entities/news/News.kt
package com.example.sportapp.support2026.features.news.domain.entities.news

import java.time.LocalDateTime

data class News(
    val id: Int,
    val dateTime: LocalDateTime,
    val sport: String,
    val imageUrl: String,
    val text: String,
    val title: String
)

File: SportAppServer/SportAppServer/SportAppServer/Support2026/Domain/Feature/NewsFeature/Entities/News.cs
namespace SportAppServer.Support2026.Domain.Feature.NewsFeature.Entities
{
    public class News
    {
        public int Id { get; set; }
        public DateTime DateTime { get; set; }
        public string Sport { get; set; }
        public string Title { get; set; }
        public string ImageId { get; set; }
        public string ArticleText { get; set; }
        public string TextAfterLemmatize { get; set; }
    }

}
